### PR TITLE
views: access control

### PR DESCRIPTION
**Pull request by lnielsen**
* NEW Adds customizable access control to record views. Allow
  configuring different permissions per endpoint.

Signed-off-by: Lars Holm Nielsen <lars.holm.nielsen@cern.ch>

### CODE REVIEW

**Comment by lnielsen** [comment]
See also inveniosoftware/invenio-records#71

File: examples/app.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def load():
 
     # Record 1 - Live record
     with db.session.begin_nested():
-        rec_uuid = uuid.uuid4()
+        rec_uuid = '9107e6ef-fea4-4971-ae2f-934d2fdcaa34'
```
Comment (jirikuncar) [inline examples/app.py]: shall we choose something artificial like: `deadbeef-1234-5678-ba11-b100dc0ffee5'

Comment (lnielsen) [inline examples/app.py]: We can't, since it has to be a valid UUID.

Comment (lnielsen) [inline examples/app.py]: I retract my statement :-)

```python
>>> from uuid import UUID
>>> UUID('deadbeef-1234-5678-ba11-b100dc0ffee5')
UUID('deadbeef-1234-5678-ba11-b100dc0ffee5')
```

**Comment by lnielsen** [comment]
Ping @nharraud: Here you can see the access control example for Records-UI.

**Comment by lnielsen** [comment]
Depends on new release of Invenio-Records with permissions integrated.

**Pull request merged**